workflow/rtma3d_pbspro_conus.xml

```diff
@@ -18,21 +18,25 @@
 <!--<!ENTITY CAP_RUN_ENVIR	""> -->
 <!ENTITY model		"&RUN;">
 <!ENTITY HRRRDAS_BEC     "1">
+<!ENTITY READIN_LOCALIZATION     "TRUE">
 
 <!ENTITY MACHINE	"cray">
 <!ENTITY machine	"&MACHINE;">
 
 <!ENTITY SCHEDULER      "pbspro">
+
+<!--- ENTITY ExpName is used to specify the testing or experiment name, and can be used in  the defination of PATH names. -->
+<!ENTITY ExpName        "">
 <!-- Variables Defined by absolute paths -->
 
-<!ENTITY ptmp_base	"/lfs/h2/emc/ptmp/&USER;/3DRTMA_HRRR_CONUS_NEW_WRKDIR">
+<!ENTITY ptmp_base	"/lfs/h2/emc/ptmp/&USER;/3DRTMA_HRRR_CONUS&ExpName;_WRKDIR">
```

Comment:
Suggest putting an underscore before ExpName, i.e., 3DRTMA_HRRR_CONUS_&ExpName;_WRKDIR
Same for NWROOT and LOG_DIR below.

Reply:
@MatthewMorris-NOAA 
Hi Matthew,

I agree with you. Actually in the beginning I put the "_" sign before ExpName. Then I found that if I did not define ExpName, then there would be an extra underscore sign "_" in all the directory name. 

So using "Expname" is kind of clumsy and is easy to cause problem, so I am going to remove this "ExpName" in the xml file. Then users could define the "ptmp_base", "NWROOT" and "LOG_DIR" for their own experimental systems.

@AnnetteGibbs-NOAA @EdwardColon-NOAA, what do you think about this?

Thanks,
-Gang